src/main/java/com/google/sps/server/CreateCalendar.java

```diff
@@ -0,0 +1,55 @@
+/*
+ * Copyright (c) 2010 Google Inc.
+ * 
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License. You may obtain a copy of the License at
+ * 
+ * http://www.apache.org/licenses/LICENSE-2.0
+ * 
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package com.google.sps.server;
+
+import com.google.api.services.calendar.model.CalendarList;
+import com.google.api.services.calendar.model.CalendarListEntry;
+
+import java.util.Date;
+import java.util.ArrayList;
+import java.util.List;
+
+import java.io.IOException;
+
+class CreateCalendar {
```

Comment:
Similar to my other comment, class names should generally not be verbs. For this case specifically, this `createCalendar` method doesn't need to be in its own class. It could instead be moved to a `ScheduleCalendar` class and used in the constructor.